### DISCUSSION TITLE
Consider to reduce tensile compiling threads

### DISCUSSION THREAD

**Discussion by ZappaBoy** [rocm-arch/rocm-arch#798] [Ideas]
Hi everyone,

I had some problems compiling the `rocblas` library. The cause of the issues is due to the number of compilation threads set to the maximum. Personally, in this way, the compilation would stop and the system would start unresponsive and make the system unusable. I solved this by decreasing the number of threads by editing the `PKGBUILD` file during installation via the AUR package manager (yay). 

It probably only happened to me, but I wanted to point out the issue in case it happened to others.

Possible solution: You might consider not using all the threads by decreasing by 2 the value of the threads in the variable `_tensile_threads`  in the `PKGBUILD` from  `_tensile_threads="$(nproc)"` to `_tensile_threads="$(($(nproc)-2))"` or even dividing by 2

**Comment by acxz** [comment]
Related to: https://github.com/rocm-arch/rocm-arch/discussions/783
